docs/changelog.rst

```diff
@@ -5,7 +5,53 @@ Changelog
 Development version (|development_version|)
 ===========================================
 
-.. |development_version| replace:: 1.0.1alpha1
+.. |development_version| replace:: 1.1.0alpha1
+
+.. _release-1.1.0alpha1:
+
+1.1.0alpha1
+===========
+
+Dependency Changes
+------------------
+
+Many documentation projects depend on ``sphinx-rtd-theme`` without specifying a version of the theme (unpinned) while also depending on unpinned versions of Sphinx. The latest version of ``sphinx-rtd-theme`` ideally always supports the latest version of Sphinx, but this is now guaranteed.
+
+This release adds upper bounds to direct dependencies ``Sphinx``, ``docutils`` and ``Jinja2``, which will safeguard from mixing with possibly incompatible future versions of Sphinx & friends.
+
+* Sphinx versions supported: 1.6 to 5.2.x
+* ``Jinja<3.1`` (#1316)
+* ``Sphinx<6`` (#1332)
+
+
+Features
+--------
+
+* Nicer styles for <kbd> (#967)
+
+
+Fixes
+-----
+
+* Suffixes in Sphinx version caused build errors (#1345)
+* Table cells with multiple paragraphs gets wrong formatting (#289)
+* Definition lists rendered wrongly in api docs (#1052)
+* Citation not styled properly (#1078)
+
+Minor changes
+-------------
+
+* Sphinx 5.2 added to test matrix (#1348)
+* Python 3.10 added to test matrix (#1334)
+* Supplemental Docker setup for development (#1319)
+* Most of setup.py migrated to setup.cfg (#1116)
+
+
+Incompatible Changes
+--------------------
+
+There are no known incompatible changes in this release, but we should note to users of ``docutils`` 0.18 and 0.19 that they are not yet support until ``sphinx-rtd-theme`` 1.2.0 is released.
```

Comment:
Less words for clarity, "next version" in case we decide to do 2.0 release instead.

```suggestion
There are no known incompatible changes in this release. Support for ``docutils`` versions 0.18 and 0.19 are scheduled for our next release.
```